features/step_definitions/brief_response_steps.rb

```diff
@@ -44,6 +45,11 @@
   page.visit("#{dm_frontend_domain}#{url}")
 end
 
+Given /^I click the 'Tell us who won this contract' link for that brief$/ do
+  url = "/buyers/frameworks/#{@framework_slug}/requirements/#{@lot_slug}/#{@brief_id}/award-contract"
+  page.find(:xpath, "//a[@href='#{url}']").click
+end
+
 Given /^I go to that brief overview page$/ do
   url = "/buyers/frameworks/digital-outcomes-and-specialists-2/requirements/#{@lot_slug}/#{@brief_id}"
```

Comment:
Do you think we should use the `@framework_slug` here as well?

Reply:
Yeah good shout, we can add that in later, tech debt ticket in _our_ sprint?